apps/react-storybook/stories/chat/Chat.stories.tsx

```diff
@@ -390,3 +390,48 @@ export const Customization: Story = {
         );
     }
 }
+
+const formattingOptions = ['shortdate', 'shorttime', 'yyyy-mm-dd hh:mm:ss', 'hh:mm:ss', 'longdate', 'longtime', 'invalid format'];
+
+const formattingControlMappings = Object.fromEntries( formattingOptions.map(option => [option, option]) ); 
+
+export const Formatting: Story = {
+    args: {
+        dayHeaderFormat: 'shortdate',
+        messageTimestampFormat: 'shorttime',
+    },
+    argTypes: {
+        dayHeaderFormat: {
+            control: 'select',
+            options: [ ...formattingOptions ],
```

Comment:
Can we avoid destructuring here, using:

options: formattingOptions, 
```suggestion
            options: formattingOptions,
            mapping: formattingControlMappings
```
?

Reply:
f8b3e8f3e8691909b04ae2609789c32a64a93fce